openapi.yaml

```diff
@@ -4972,6 +5025,7 @@ paths:
     delete:
       tags:
         - Users
+      x-sdk-operation: deleteAll
```

Comment:
This operation actually sits under `DELETE /workspace/users`, therefore it is a delete all operation.